src/docs/contributing.md

```diff
@@ -0,0 +1,152 @@
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### Before Submitting A Bug Report
+
+* **Check the [discussions](https://github.com/mapping-commons/sssom/discussions)** for a list of common questions and problems.
+* **Decide whether the issue should be reported in the tracker for the [SSSOM data model](https://github.com/mapping-commons/sssom/issues) or the tracker for the [SSSOM toolkit](https://github.com/mapping-commons/sssom-py/issues)**.
+* **Perform a [cursory search](https://github.com/mapping-commons/sssom/issues)** to see if the problem has already been reported. If it has **and the issue is still open**, add a comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Bug Report or Feature request?
+
+Bugs and feature requests are tracked as [GitHub issues](https://guides.github.com/features/issues/). After you've determined which repository your bug or feature is related to, create an issue on that repository providing the information required by [the appropriate template](https://github.com/mapping-commons/sssom/issues/new/choose).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem/requests.
+* **Describe the exact steps which reproduce the problem** in as many details as possible. For example, start by explaining how you started SSSOM, e.g. which command exactly you used in the terminal, or how you started SSSOM otherwise. When listing steps, **don't just say what you did, but explain how you did it**. For example, if you moved the cursor to the end of a line, explain if you used the mouse, or a keyboard shortcut or an SSSOM command, and if so which one?
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+
+Include details about your configuration and environment:
+
+* **Which version of SSSOM toolkit/model are you using?** You can get the exact version by running `sssom --version` in your terminal
+* **What's the name and version of the OS you're using**?
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to SSSOM? You can start by looking through these `beginner` and `help-wanted` issues:
+
+* [Beginner issues][beginner] - issues which should only require a few lines of code, and a test or two.
```

Comment:
fix links with round parentheses on this and the next line

Reply:
Actually - these are github variables: https://stackoverflow.com/questions/24580042/github-markdown-are-macros-and-variables-possible